README.md

```diff
@@ -26,14 +26,11 @@ allowing users to integrate with their existing tooling and processes.
 * **Repo configuration and security:** Simplify configuration and management of security settings and policies across repos.
 * **Proactive security enforcement:** Continuously enforce best practice security configurations by setting granular policies to alert only or auto-remediate.
 * **Artifact attestation:** Continuously verify that packages are signed to ensure they’re tamper-proof, using the open source project Sigstore.
-* **Dependency management:** Manage dependency security posture by helping developers make better choices and enforcing controls. Minder is integrated with [Trusty by Stacklok](https://trustypkg.dev) to enable policy-driven dependency management based on the risk level of dependencies.
+* **Dependency management:** Manage dependency security posture by helping developers make better choices and enforcing controls. Minder is integrated with [OSV](https://osv.dev/) and [Trusty](https://trustypkg.dev) to enable policy-driven dependency management based on the risk level of dependencies.
 
-## Minder Cloud
+## Public Instance
 
-Stacklok, the company behind Minder, provides a [free-to-use SaaS version of Minder](https://cloud.stacklok.com/) that includes a UI (for public repositories only). You can access Minder Cloud documentation [here](https://docs.stacklok.com/minder).
-
-Note that it's not possible to register private repositories. If you'd like to use Minder with private repositories,
-feel free to [contact us](mailto:hello@stacklok.com)! We'd be thrilled to help you out.
+Stacklok, a company supporting Minder, provides a free-to-use SaaS version of Minder at api.stacklok.com. This instance is available for public repositories only.
```

Comment:
`cloud.stacklok.com`?

Reply:
Oh, I think that I see where you're going with this. I have a bit of trouble parsing this because it feels like something that I should visit or something. Maybe ...

```suggestion
Stacklok, a company supporting Minder, provides a free-to-use public instance of Minder. This is the default instance used when you use the `minder` CLI. This instance is available for public repositories only.
```

Reply:
The thinking is we should avoid directing folks to a propriety UI, within a community set of docs. api.stacklok.com is near on identical to running it locally.